### PR TITLE
adding an infobox

### PR DESCRIPTION
**Pull request by thorbenN65**
So I rebuilt the modal to work without JavaScript and deliver a better experience on mobile devices.

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -59,9 +61,93 @@ body {
     font-weight: bold;
     cursor: pointer;
     transition: 0.25s;
+
```
Comment (fedus) [inline src/css/main.css]: Superfluous line break

File: src/css/main.css [hunk 2]
```diff
@@ -10,7 +10,9 @@ body {
     margin: 0;
     padding: 0;
 }
-
+h1 {
+    font-size: 0.75rem;
+}
```
Comment (fedus) [inline src/css/main.css:16]: Line break missing

File: src/css/main.css [hunk 3]
```diff
@@ -59,9 +61,93 @@ body {
...
 .generate-button:hover {
     color: #000000;
     background-color: #FFFFFF;
-}
\ No newline at end of file
+}
+
+.modalposition {
+    text-align: center;
+}
```
Comment (fedus) [inline src/css/main.css:75]: Line break missing

File: src/css/main.css [hunk 4]
```diff
@@ -59,9 +61,93 @@ body {
...
+    padding: 5px 15px;
+    font-weight: bold;
+    cursor: pointer;
+    transition: 0.25s;
+    text-decoration: none;
+    display: inline;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+  }
```
Comment (fedus) [inline src/css/main.css:92]: Line break missing

File: src/css/main.css [hunk 5]
```diff
@@ -59,9 +61,93 @@ body {
...
+  .modalbutton:hover {
```
Comment (fedus) [inline src/css/main.css]: Odd indentation

File: src/css/main.css [hunk 6]
```diff
@@ -59,9 +61,93 @@ body {
...
+    background:black;
+    color: white;
+  }
```
Comment (fedus) [inline src/css/main.css]: Line break missing

File: src/css/main.css [hunk 7]
```diff
@@ -59,9 +61,93 @@ body {
...
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(0, 0, 0, 0.7);
+    transition: opacity 0.5s;
+    visibility: hidden;
+    opacity: 0;
+  }
```
Comment (fedus) [inline src/css/main.css]: Etc etc line breaks and odd indentation :)

File: src/css/main.css [hunk 8]
```diff
@@ -59,9 +61,93 @@ body {
...
+  @media screen and (max-width: 700px){
+    .box{
+      width: 70%;
+    }
+    .popup{
+      width: 70%;
+      font-size: 00.5em;
+
+    }
+  }
```
Comment (fedus) [inline src/css/main.css]: No newline at end of file

File: src/index.html [hunk 9]
```diff
@@ -23,10 +24,43 @@
 
     </head>
     <body class="flex-container" x-data="carFestival()">
-        <div class="excuse-top">I need a new car because ...</div>
+        <div class="excuse-top"><h1>I need a new car because ...</h1></div>
         <div class="excuse-bottom" x-show="!showGuessButton" x-text="currentExcuse"></div>
         <div class="generate-button-container" x-show="showGuessButton">
-            <button type="button" class="generate-button" x-on:click="guess">Generate excuse</button>
+            <button type="modalbutton" class="generate-button" x-on:click="guess">Generate excuse</button>
```
Comment (fedus) [inline src/index.html]: You changed the existing button's type to "modalbutton" ... that doesn't seem HTML compliant ;)

File: src/index.html [hunk 10]
```diff
@@ -93,5 +127,7 @@
                 },
             };
         }
+
```
Comment (fedus) [inline src/index.html]: Odd linebreaks ;)

**Review by fedus — CHANGES_REQUESTED** [review]
All in all good job!

File: src/index.html (hunk 9 above)
Comment (thorbenN65) [reply src/index.html]: Html...schmahtml

**Review by fedus — APPROVED** [review]

**Pull request merged**